core/src/edu/cornell/gdiac/temporary/GameplayController.java

```diff
@@ -51,71 +55,57 @@ public class GameplayController {
 	private static final float MIN_STAR_OFFSET = -3.0f;
 	/** The maximum velocity offset (+ shell velocity) of a newly created star */
 	private static final float MAX_STAR_OFFSET = 3.0f;
-	/** The amount of health gained when hitting a note weakly */
-	private static final int WEAK_HIT_HEALTH = 1;
-	/** The amount of health gained when hitting a note strongly */
-	private static final int STRONG_HIT_HEALTH = 2;
-	/** The amount of health lost when missing a note */
-	private static final int MISS_HIT_HEALTH = 2;
+
+
 	/** Maximum amount of health */
 	final int MAX_HEALTH = 30;
 
 	/** Number of band member lanes */
 	int NUM_LANES;
 
-	/** Number of notes present in repeating rhythm */
-	private static final int NUM_NOTES = 60;
-	/** Note count for the display in window corner */
-	private int shellCount;
-
-	/** health of each band member */
-	private int[] health;
 
 	// List of objects with the garbage collection set.
 	/** The currently active object */
 	private Array<GameObject> objects;
 	/** The backing set for garbage collection */
 	private Array<GameObject> backing;
 
-	HashMap<Integer, Note> noteCoords = new HashMap<>();
 
 	/**
 	 * Index of the currently active band member
 	 */
-	int currentLane;
+	int activeBM;
 
 	/**
-	 * The base hp increment. Hp will incrememnt on a destroyed note by the product of this value and its hit status
+	 * Array of bandmembers
 	 */
-	final int recovery = 3;
-
-	private void setCoords(float width, float height) {
-		// note appears every two seconds if we have a 30 second loop
-		// 1800
-		for (int i = 0; i < NUM_NOTES; i++) {
-			Note s = new Note(i%4, Note.NType.BEAT);
-			s.setX(width/8 + (i % 4) * width/4);
-			s.setTexture(catNoteTexture);
-			s.setY(height);
-			s.setVX(0);
-			s.setVY(-5f);
-			noteCoords.put(i * 30, s);
-		}
-	}
+	BandMember[] bms;
```

Comment:
Please rename this to bandMembers because seeing `bms` is honestly hard to read. 

Also as a discussion point, this will get moved to a level class, correct?

Reply:
Will do